src/style-sheets/components/_meet-the-team-card.scss

```diff
@@ -0,0 +1,101 @@
+.meet-the-team-card {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    width: 100%;
+    padding: 5% 0;
+    margin-bottom: 2.5714em;
+    font-family: $olBaseFont;
+
+    @include small('up') {
+      width: 95%;
+      padding: 4% 0;
+    }
+
+    @include large('up') {
+      margin-bottom: 3.4286em;
+    }
+
+    .card {
+      &-content {
+        display: grid;
+        grid-template-columns: 1fr .4fr 4fr;
+        grid-template-rows: .2fr .2fr .2fr;
+        grid-template-areas:
+            "img . fullname"
+            "img . member-role"
+            "img . location";
+  
+        width: 90%;
+        margin-bottom: 1.7143em;
+  
+        @include query($screen-sm, $screen-md) { 
+          width: 87%;
+        }
+  
+        @include medium('up') {
```

Comment:
Hmm, looks like each separate part of the card has its own media queries. Why not have one media query that includes the changes for all the parts? Is this for the sake of clarity?

Reply:
This was discussed in our discord chats but having the query in the part that we are changing adds to the readability a bit. It makes more sense when you implement it since you don't have to dig through the huge chunk of media query